infra/main.bicep

```diff
@@ -398,6 +398,7 @@ module web './app/web.bicep' = if (hostingModel == 'code') {
       AZURE_OPENAI_API_VERSION: azureOpenAIApiVersion
       AZURE_OPENAI_STREAM: azureOpenAIStream
       AZURE_OPENAI_EMBEDDING_MODEL: azureOpenAIEmbeddingModel
+      AZURE_RESOURCE_GROUP: rgName
```

Comment:
If we are going to make this change to every app service - we could move this down to `/infra/core/host/appservice.bicep`

Reply:
Done